### DISCUSSION TITLE
Ticwatch pro as standalone with G6

### DISCUSSION THREAD

**Discussion by avi532** [Navid200/xDrip#105] [Dexcom]
I recently switched from libre to dexcom g6. i am using lg phone with xdrip and a ticwatch pro. however when i try to force collect so the watch can be used as standalone, i don't receive any data on phone or watch. has anyone had this issue? is there a different way to make watch as standalone when using dexcom? On libre it workd fine

**Comment by Navid200** [comment]
Is anything relevant here?
https://github.com/NightscoutFoundation/xDrip/blob/master/Documentation/WatchGuide.md

**Reply by avi532** [reply]
Thanks i went through that already, didn't help. I wonder if others have had this issue with dexcom g6

**Comment by Navid200** [comment]
We have discussions now opened in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

I am looking for a way to transfer the open discussions from here to there. But, I'm not sure I'll succeed. In that case, it may be easier to open new discussions there.
